spec/order_repository_spec.rb

```diff
@@ -0,0 +1,57 @@
+  require 'order_repository'
+
```

Comment:
With the order repository spec, I like how Harry structured his code. his methods and variables are very clear